wgsl/index.bs

```diff
@@ -659,9 +659,8 @@ Triggered diagnostics [=behavioral requirement|will=] be processed as follows:
          * The error is a [=pipeline-creation error=] if the diagnostic was triggered at [=pipeline creation=] time.
 6. If processing during [=shader module creation=] time, the remaining diagnostics populate the
     {{GPUCompilationInfo/messages}} member of the WebGPU {{GPUCompilationInfo}} object.
-7. If processing during [=pipeline creation=], the [=severity/error=] diagnostics are reported
-    in the nearest enclosing WebGPU [=GPU error scope=].
-    Issue: TODO: Adjust for [#3682](https://github.com/gpuweb/gpuweb/issues/3682).
+7. If processing during [=pipeline creation=], [=severity/error=] diagnostics
```

Comment:
This isn't a functional change, but I wonder if there is a future where there are warnings generated during pipeline creation. There doesn't seem to be a mechanism to surface them. Not sure it needs solved now though.

Reply:
There definitely is such a future, but we don't have it now. We have considered this and planned to possibly communicate error/warning messages via new attributes on GPURenderPipeline/GPUComputePipeline/GPUPipelineError.